<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>1171</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="4">
      <widget class="QPushButton" name="rendButton">
       <property name="text">
        <string>REND</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="frontButton">
       <property name="text">
        <string>FRONT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>BACK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="emplaceButton">
       <property name="text">
        <string>EMPLACE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="capacityButton">
       <property name="text">
        <string>CAPACITY</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="emplace_back_Button">
       <property name="text">
        <string>EMPLACE_BACK</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="insertButton">
       <property name="text">
        <string>INSERT</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="assignButton">
       <property name="text">
        <string>ASSIGN</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="atButton">
       <property name="text">
        <string>AT</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>CLEAR</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="rbeginButton">
       <property name="text">
        <string>RBEGIN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="max_size_Button">
       <property name="text">
        <string>MAX_SIZE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pop_back_Button">
       <property name="text">
        <string>POP_BACK</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="beginButton">
       <property name="text">
        <string>BEGIN</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="push_back_Button">
       <property name="text">
        <string>PUSH_BACK</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="eraseButton">
       <property name="text">
        <string>ERASE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="cbeginButton">
       <property name="text">
        <string>CBEGIN</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="emptyButton">
       <property name="text">
        <string>EMPTY</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="endButton">
       <property name="text">
        <string>END</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="dataButton">
       <property name="text">
        <string>DATA</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="reserveButton">
       <property name="text">
        <string>RESERVE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="resizeButton">
       <property name="text">
        <string>RESIZE</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QPushButton" name="sizeButton">
       <property name="text">
        <string>SIZE</string>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QPushButton" name="swapButton">
       <property name="text">
        <string>SWAP</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:700; color:#813d9c;&quot;&gt;Демонстрация работы класса &amp;quot;Вектор&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="function_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;Функции вектора:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="vector_content_textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="view_vector_content_Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>581</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Посмотреть содержимое вектора</string>
    </property>
   </widget>
   <widget class="QLabel" name="function_result_label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:700; color:#ed333b;&quot;&gt;Результат выполнения функции:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="function_result_textBrowser">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="help_for_user_Button">
    <property name="geometry">
     <rect>
      <x>956</x>
      <y>60</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ПРЕДНАЗНАЧЕНИЕ ФУНКЦИЙ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vectorButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Заполнить вектор!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="class_pair_Button">
    <property name="geometry">
     <rect>
      <x>956</x>
      <y>15</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Класс PAIR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1191</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
